src/test/java/org/opentripplanner/model/TimetableSnapshotTest.java

```diff
@@ -105,12 +113,21 @@ public void testUpdate() {
         TripDescriptor.Builder tripDescriptorBuilder = TripDescriptor.newBuilder();
 
         tripDescriptorBuilder.setTripId("1.1");
-        tripDescriptorBuilder.setScheduleRelationship(ScheduleRelationship.CANCELED);
+        tripDescriptorBuilder.setScheduleRelationship(ScheduleRelationship.SCHEDULED);
```

Comment:
Could you perhaps explain why this is necessary? I think I'm missing something here. Should this not be a separate test case?

Reply:
This tests through the `Timetable` class' `createUpdatedTripTimes` method. We don't use it for handling CANCELED updates as it's done through `TimetableSnapshotSource` class directly. Therefore, I removed the support for handling CANCELED trips in `createUpdatedTripTimes` and updated the tests that used it.

Reply:
Ok, thanks for the explanation.